src/application/services/themeService.ts

```diff
@@ -0,0 +1,37 @@
+import { ref } from 'vue';
+
+const currentBaseTheme = ref((localStorage.getItem('baseTheme') ?? '') || 'theme-base-pure');
+const currentAccentTheme = ref((localStorage.getItem('accentTheme') ?? '') || 'theme-accent-classic');
```

Comment:
according to @neSpecc's architecture, you can't just access local storage from application service 🥲

you need to create a storage, then access it from repository, access repository from domain service, and access service from application service. 

you can examine how storing user data is implemented to understand better